packages/cdktf-cli/bin/cmds/helper/terraform-check.ts

```diff
@@ -4,19 +4,13 @@ import { SynthesizedStack } from "../../../lib/synth-stack";
 import { existsSync } from "fs-extra";
 import * as path from "path";
 import { AbortController } from "node-abort-controller"; // polyfill until we update to node 14
+import { logger } from "../../../lib/logging";
 
-const MIN_SUPPORTED_VERSION = "0.13.0";
+const MIN_SUPPORTED_VERSION = "1.0.0";
```

Comment:
Where is this used? We should add at least a line to our changelog (about init requiring TF 1.1 now if using TFC).
If there is no change for existing projects, I'd be fine to release this as a patch even. Else we should only include this changed requirement with the next minor (i.e. 0.13).

Reply:
We run this as on init, login and whenever a react component executes. I can also remove this, but I think we already require users to be on TF 1.0 or above, don't we?

Reply:
Yeah, but enforcing this is new.

Reply:
Then let's remove this commit (I made a separate one for this line)

Reply:
I checked the code and it does only warn but not fail, so keeping it would be fine with me

Reply:
However, we should document that >1.1 is forced for init now, if users use remote state.